### PR TITLE
Fix remote DUD and support multiple DUDs

### PR DESCRIPTION
**Pull request by dzedro**
- Verification run: http://10.100.12.155/tests/12093#step/bootloader/13

### CODE REVIEW

File: lib/bootloader_setup.pm [hunk 1]
```diff
@@ -638,14 +638,17 @@ sub specific_bootmenu_params {
...
-    if (get_var("DUD")) {
-        my $dud = get_var("DUD");
-        if ($dud =~ /http:\/\/|https:\/\/|ftp:\/\//) {
-            push @params, ("dud=$dud", 'insecure=1');
-        }
-        else {
-            push @params, ("dud=" . data_url($dud), 'insecure=1');
+    if (get_var('DUD')) {
+        my @duds = split(/,/, get_var('DUD'));
+        while (defined(my $dud = shift @duds)) {
```
Comment (kravciak) [inline lib/bootloader_setup.pm]:
```suggestion
        foreach $dud (@duds) {
```

File: lib/bootloader_setup.pm [hunk 2]
```diff
@@ -638,14 +638,17 @@ sub specific_bootmenu_params {
...
+            if ($dud =~ /^http\:\/\/|^https\:\/\/|^ftp\:\/\//) {
```
Comment (kravciak) [inline lib/bootloader_setup.pm]:
```suggestion
            if ($dud =~ /^(http|https|ftp):\/\//) {
```

File: lib/bootloader_setup.pm [hunk 3]
```diff
@@ -638,14 +638,17 @@ sub specific_bootmenu_params {
...
+                push @params, ("dud=$dud");
```
Comment (kravciak) [inline lib/bootloader_setup.pm]:
```suggestion
                push @params, "dud=$dud";
```

File: lib/bootloader_setup.pm [hunk 4]
```diff
@@ -638,14 +638,17 @@ sub specific_bootmenu_params {
...
+            }
+            else {
+                push @params, ("dud=" . data_url($dud));
```
Comment (kravciak) [inline lib/bootloader_setup.pm]:
```suggestion
                push @params, "dud=" . data_url($dud);
```

File: lib/bootloader_setup.pm [hunk 5]
```diff
@@ -638,14 +638,17 @@ sub specific_bootmenu_params {
...
+            }
         }
+        push @params, ('insecure=1');
```
Comment (kravciak) [inline lib/bootloader_setup.pm]:
```suggestion
        push @params, 'insecure=1';
```

File: lib/bootloader_setup.pm (hunk 1 above)
Comment (dzedro) [reply lib/bootloader_setup.pm]: missing `my`

**Pull request merged**